spacy/scorer.py

```diff
@@ -172,12 +172,8 @@ def score(self, doc, gold, verbose=False, punct_labels=("p", "punct")):
                     self.ner_per_ents[ent.label_].fp += 1
                 else:
                     cand_ents.add((ent.label_, first, last))
-                    current_ent[ent.label_].add(
-                        tuple(x for x in cand_ents if x[0] == ent.label_)
-                    )
-                    current_gold[ent.label_].add(
-                        tuple(x for x in gold_ents if x[0] == ent.label_)
-                    )
+                    [current_ent[ent.label_].add(x) for x in cand_ents if x[0] == ent.label_]
```

Comment:
Pretty minor style thing, but I try to avoid list comprehensions that aren't assigned to anything.

```suggestion
                    current_ent[ent.label_].update([x for x in cand_ents if x[0] == ent.label_])
```